maker/src/routes.rs

```diff
@@ -119,3 +121,22 @@ pub async fn post_open_channel(
 
     Ok(Json(OpenChannelResponse { funding_txid }))
 }
+
+#[rocket::post("/invoice/send/<invoice>")]
```

Comment:
I'd also change this `send` to `pay`.
Note, this API is open to the public. I don't mind for the tournament but thought to point it out.

Reply:
I've created a new ticket out of this so we won't forget after the tournament.

Reply:
> I'd also change this `send` to `pay`. Note, this API is open to the public. I don't mind for the tournament but thought to point it out.

that's true. I don't think adding authentication when only using testnet makes much sense, so I didn't add it yet.
However, I added a ticket to ensure we will not forget about it later: 
https://github.com/itchysats/10101/issues/259